### PR TITLE
Logic to show comments in the company Evaluation!

### PR DESCRIPTION
**Pull request by sofipacifico**
**Done
*Added logic to show the comments of the evaluator and califications in the company evaluation!
*Change div for textarea
*Improve textarea style

### CODE REVIEW

**Comment by andresmoschini** [comment]
Just for curiosity, what kind of markup are you using?

**Comment by andresmoschini** [comment]
I asked because maybe you could be interested in _Markdown_. I think that it is similar to what you was doing, it is really simple, plain text friendly and automatically convertible to **HTML** in the majority of the important sites, like _GitHub_, _Trello_, etc.

See more information in: 

* [The Markdown Tutorial](http://eherrera.net/markdowntutorial/)
* [GitHub Flavored Markdown](https://help.github.com/articles/github-flavored-markdown/)
* [Original author site](http://daringfireball.net/projects/markdown/)

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js [hunk 1]
```diff
@@ -68,6 +64,20 @@
...
 
+        if (this.userView == 3) {
+            var comments = "";
+            for (var i in self.califications) {
+                var calification = self.califications[i];
+                if ((calification.owner == 1 || calification.owner == 2) && calification.comments() != null) {
+                    if (comments) {
+                        comments += "\n\n";
+                    }
+                    comments += calification.evaluatorEmployee + ': ' + calification.comments();
```
Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js]: Maybe `Array.join()` could be better.

**Comment by andresmoschini** [comment]
Nice! 

I have only proposed a little enhancement.

BTW, I have sent you and invitation to `collaborators`, if you accept it you will be available to merge for yourself.

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js (hunk 1 above)
Comment (joseguti84) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js]: I agree. I will change that in the next PR

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js [hunk 2]
```diff
@@ -68,6 +64,20 @@
             return  calification.evaluatorEmployee == self.userLogged;
         }).comments;
 
+        if (this.userView == 3) {
```
Comment (AlphaGit) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js]: I'd suggest avoiding magic numbers. At the very least we can setup "constants" in JS and use those.

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js [hunk 3]
```diff
@@ -68,6 +64,20 @@
...
+            var comments = "";
+            for (var i in self.califications) {
+                var calification = self.califications[i];
+                if ((calification.owner == 1 || calification.owner == 2) && calification.comments() != null) {
```
Comment (AlphaGit) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js]: Ditto here.

**Pull request merged**